src/Atmos/Model/moisture.jl

```diff
@@ -73,8 +73,9 @@ vars_aux(::EquilMoist,FT) = @vars(temperature::FT, θ_v::FT, q_liq::FT)
 
 @inline function atmos_nodal_update_aux!(moist::EquilMoist, atmos::AtmosModel,
                                          state::Vars, aux::Vars, t::Real)
+  FT=eltype(state)
   e_int = internal_energy(moist, atmos.orientation, state, aux)
-  TS = PhaseEquil(e_int, state.ρ, state.moisture.ρq_tot/state.ρ, moist.maxiter)
+  TS = PhaseEquil(e_int, state.ρ, state.moisture.ρq_tot/state.ρ, FT(1))
```

Comment:
I would suggest waiting for #711 if you want control over max iterations. Using `FT(1)` will break (MethodError) since this is expected to be an integer. If you meant for this to be the tolerance, `tol` comes after `maxiter`.

Reply:
Oddly enough it didn't break. though this isn't updated with regards to the current master. The case is still a large work in progress